modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -52,6 +52,25 @@ Even if your projects have not been identified as requiring action, you can proa
 [fix-gov-issues]
 == Fix Governance Conformance Issues
 
+When your API fails conformance to rulesets your organization validates against, you get an Email notification:
+
+. Click *Fix me* on your email to be redirected to the Exchange API asset Conformance Status page.
+.. If the latest version of your API has an editable branch, a pop-up appears where you can see the missing rulesets. Click *Add all rulesets*. +
+_Result_: The rulesets are added to your project.
+.. If the latest version of your API doesn't have an editable branch, a pop-up appears where you can select an editable branch on your project, or create a new branch from your current version to add the missing rulesets. 
+When you have your branch, you can add your rulesets manually:
```

Comment:
This is not correct, when user selects a branch or creates a new one,  a pop-up appears where you can see the missing rulesets. User can  Click *Add all rulesets* but also has the possibility to click on Edit Criteria to change search for rulesets that applied if the tags/categories or Lifecycle status of the asset changed